aws/bootstrap.tf

```diff
@@ -242,6 +238,20 @@ resource "null_resource" "bootstrap" {
     agent = "${local.agent}"
   }
 
+  }
```

Comment:
This additional brace needs to be deleted.

Reply:
@sesh-kebab  thanks!